proto/zitadel/user/v2beta/password.proto

```diff
@@ -55,3 +55,31 @@ enum NotificationType {
   NOTIFICATION_TYPE_Email = 1;
   NOTIFICATION_TYPE_SMS = 2;
 }
+
+message SetPassword {
+  oneof password_type {
+    Password password = 7;
+    HashedPassword hashed_password = 8;
+  }
+  oneof verification {
+    string current_password = 3 [
+      (validate.rules).string = {min_len: 1, max_len: 200},
+      (google.api.field_behavior) = REQUIRED,
+      (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+        min_length: 1;
+        max_length: 200;
+        example: "\"Secr3tP4ssw0rd!\"";
+      }
+    ];
+    string verification_code = 4 [
+      (validate.rules).string = {min_len: 1, max_len: 20},
+      (google.api.field_behavior) = REQUIRED,
+      (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+        min_length: 1;
+        max_length: 20;
+        example: "\"SKJd342k\"";
+        description: "\"the verification code generated during password reset request\"";
+      }
+    ];
+  }
+}
```

Comment:
Message field numbering should start from 1.

Reply:
Changed